app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -142,7 +143,8 @@ def vm_create_evm_managed_snapshot(vm)
       snap_svc.get(ssa_snap_name, resource_group)
     rescue ::Azure::Armrest::NotFoundException, ::Azure::Armrest::ResourceNotFoundException => err
       begin
-        snap_svc.create(ssa_snap_name, resource_group, snap_options)
+        response = snap_svc.create(ssa_snap_name, resource_group, snap_options)
+        raise "Maximum snapshot wait time exceeded" unless snap_svc.wait(response.response_headers, SSA_SNAPSHOT_WAIT_TIME) == "Succeeded"
```

Comment:
I'd change this to `=~ /^succe/i` because, depending on the operation, azure returns "Success" or "Succeeded", and I don't remember where and/or why.

Reply:
@djberg96 cool - thanks for the heads up.